### DISCUSSION TITLE
Selecting the provider for each widget separately

### DISCUSSION THREAD

**Discussion by SamLab** [breezy-weather/breezy-weather#1021] [Ideas]
### Describe your idea

Hello.
I have been using your application for a long time and I like it very much, but one very useful feature is missing: can you make it possible to select a provider for each widget - so you can see several forecasts on the desktop and compare them to determine the weather accurately. Especially interested in this feature on hourly and weekly widgets.
Thanks in advance.

### Other details

No provider gives an accurate forecast, so on the desktop I would like to display the same widgets, but with different providers, so that on the basis of several, you can make an average forecast.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.2 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Duplicate of #391?

**Reply by SamLab** [reply]
No, the thread you mentioned suggests selecting a location for each widget, and I suggest selecting a weather provider for each widget

**Reply by papjul** [reply]
If you mean a weather source, it's tied to the location, and you can add the same location with different weather sources, so what's the difference?

**Reply by papjul** [reply]
![IMG_20240430_154514](https://github.com/breezy-weather/breezy-weather/assets/1449159/90282930-401a-422a-9ca1-f692f5e09936)

**Reply by SamLab** [reply]
Yes, you can select in the program, but when you create a widget, it doesn't let you select a location and defaults to creating a widget with the first one on the list.

**Reply by papjul** [reply]
Thanks, closing as duplicate
